deploy/fio.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: batch/v1
```

Comment:
There are a lot of community users who make use of the samples/examples available in various github repos to run their test & probably prod workloads.

Having said that, can we have below:
- Merge all the related yamls as a single yaml file
- Put yaml comments that explain why, what, etc

Reply:
sure, these are test yamls which I use for my testing .